docs/proof-trace.md

```diff
@@ -21,7 +21,9 @@ have rendered this unnecessary, but the change hasn't been implemented yet.
 
 Here is a BNF styled description of the format:
 ```
-proof_trace ::= event*
+proof_trace ::= header event*
+
+header      ::= "HINT" <4-byte version number>
```

Comment:
Could we have this be a semantic version like in the binary KORE format (which uses 3 16-bit unsigned integers for `major . minor . patch`)

Reply:
See comment below, in fact

Reply:
@dwightguth asked for the single integer over the semantic version. I think I agree with doing it this way, the format shouldn't really go through any radical changes, and there probably wont be any widespread use of it outside of a couple of our tools.

Reply:
In that case it's fine by me!